schemas/oic.sec.dostype.json

```diff
@@ -19,6 +19,7 @@
         },
         "p": {
           "type": "boolean",
+          "readOnly": true,
```

Comment:
I'm not sure an element of a Property can be read-only in a meaningful way.  What is meant to happen if a POST includes the "p" element of the "dos" Property?  Is "p" ignored but the rest of the Property (e.g. "s") Updated?  Or is the POST rejected for containing a read-only element of a read/write Property?

Reply:
I agree with you, perhaps we need a separate schema to capture UPDATE schema? Please note as per the spec this is readonly.